localization.py

```diff
@@ -0,0 +1,60 @@
+import Constant
+
+CURR_LOCALIZATION = "en-US"
+
+TRANSLATION_TABLE = {
+    Constant.LOCALIZATION_SUPPORT[0]: {
+        "battery_param_report": {
+            Constant.PARAM_OPERATING_RANGE_CLASSIFIER[0]: "ALERT: Battery parameter %s is NaN.",
+            Constant.PARAM_OPERATING_RANGE_CLASSIFIER[1]: "ALERT: Battery parameter %s is not supported for safety checks.",
+            Constant.PARAM_OPERATING_RANGE_CLASSIFIER[2]: "ALERT: Battery parameter %s is out of range!",
+            Constant.PARAM_OPERATING_RANGE_CLASSIFIER[3]: "ALERT: Battery parameter %s is out of range!",
+            Constant.PARAM_OPERATING_RANGE_CLASSIFIER[4]: "ALERT: Battery parameter %s is over the prescribed limit",
+            Constant.PARAM_OPERATING_RANGE_CLASSIFIER[5]: "ALERT: Battery parameter %s is over the prescribed limit",
+            Constant.PARAM_OPERATING_RANGE_CLASSIFIER[6]: "WARNING: Battery parameter %s is approaching the prescribed "
+                                                "underlimit.",
+            Constant.PARAM_OPERATING_RANGE_CLASSIFIER[7]: "WARNING: Battery parameter %s is approaching the prescribed "
+                                                "overlimit.",
+            Constant.PARAM_OPERATING_RANGE_CLASSIFIER[8]: "INFO: Battery parameter %s is normal. Running within prescribed "
+                                                "limits. "
+        },
+        "overall_battery_report": {
+            Constant.BATTERY_STATUS_CLASSIFIER[0]: "ALERT: Battery operating outside safe parameters. Please stop usage"
+                                                   " immediately.",
+            Constant.BATTERY_STATUS_CLASSIFIER[1]: "INFO: Battery running normally."
+        }
+    },
+    Constant.LOCALIZATION_SUPPORT[1]: {
+        "battery_param_report": {
+            Constant.PARAM_OPERATING_RANGE_CLASSIFIER[0]: "ALARM: Batterieparameter %s ist NaN.",
```

Comment:
What kind of test would ensure that none of the translations were missed out?

Reply:
I am actually blanking on this. Could you help me out on this? A looping or random picker over the possible range PARAM_OPERATING_RANGE_CLASSIFIER is all I can think of...